app/src/main/java/com/example/blah/mobilestudio/breadcrumbview/BreadcrumbView.java

```diff
@@ -28,7 +28,7 @@
     private ArrayList<String> listOfElements;
 
     Paint paintColorStyle;
-    String rootPath = "";
+    public String rootPath = "";
```

Comment:
Initialise the variables in the constructor, and explicitly declare them as private/public, unless they are final variables.
eg
`private String currentPath;

public BreadCrumbView(){
    currentPath = "";
}`